README.md

```diff
@@ -1 +1,32 @@
-# burger_town
+# BurgerTown
+
+> BurgerTown is a User Interface created with ReactApp that utilizes the [Bob's Burgers API](https://www.bobsburgersapi.com/) to access all things Bob's Burgers.
+
+## Install
+
+---
+
+While in root folder
+
+```
+cd client
```

Comment:
Putting the React app entirely inside this client folder is a habit from your fullstack app that isn't necessary here. Everything in the client folder can live at the root level of the project just fine, and it probably should since this project is only a SPA